client/wfnews-war/src/main/angular/src/app/components/full-details/evac-other-info/evac-other-info.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component } from '@angular/core';
+import { AppConfigService } from '@wf1/core-ui';
+
+@Component({
+  selector: 'wfnews-evac-other-info',
+  templateUrl: './evac-other-info.component.html',
+  styleUrls: ['./evac-other-info.component.scss']
+})
+export class EvacOtherInfoComponent {
+
+  constructor(private appConfigService: AppConfigService) {
+
+  }
+
+  wildfirePreparedness() {
```

Comment:
Not important, but I think we can simplify these window.open() to one method 
ex: 
openLink(link: string) {
window.open(this.appConfigService.getConfig().externalAppConfig[link] as unknown as string, '_blank')
}
and just pass different variable with those (click) call in html file.

Reply:
Good idea!